src/Idris/Parser.idr

```diff
@@ -1397,6 +1397,8 @@ fnDirectOpt : OriginDesc -> Rule PFnOpt
 fnDirectOpt fname
     = do pragma "hint"
          pure $ IFnOpt (Hint True)
+  <|> do pragma "chaser"
```

Comment:
Why is this necessary compared to not using the pragma? I assumed every definition was `Hint False` by default?

Reply:
Actually, every definition isn't `Hint _` by default.

Reply:
Hmmm, seems that this can't be merged until `chaser` is available on the new boostrap.